repositories/metadata/postgresql/migrations/sql/0006_add_versions_ttl.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+ALTER TABLE containers
+    ADD COLUMN versions_ttl BIGINT DEFAULT -1;
```

Comment:
1) Good practice for such case is to define units in the name i.e. "versions_ttl_seconds" or "versions_ttl_hours" it would make that part more clear to understand.
2) Normally in such case first word where the second is applicable to goes in singular form so it would be "version_ttl_seconds" for example